packages/editor/src/components/post-card-panel/index.js

```diff
@@ -89,12 +103,17 @@ export default function PostCardPanel( { className, actions } ) {
 					{ actions }
 				</HStack>
 				<VStack className="editor-post-card-panel__content">
-					{ ( description || lastEditedText ) && (
+					{ ( description ||
+						lastEditedText ||
+						showPostContentInfo ) && (
 						<VStack
 							className="editor-post-card-panel__description"
 							spacing={ 2 }
 						>
 							{ description && <Text>{ description }</Text> }
+							{ showPostContentInfo && (
+								<PostContentInfo postContent={ postContent } />
```

Comment:
postContent is not used in the current component, why not select it in the underlying component instead?

Reply:
I opened a follow up: https://github.com/WordPress/gutenberg/pull/60743